category.html

```diff
@@ -0,0 +1,19 @@
+---
+title: "Category Home"
+layout: default
+permalink: /blog/category/
+---
+
+<h1>These are the categories for all of our blog posts!</h1>
+
+<p>They link to the corresponding category pages!</p>
```

Comment:
Some rewording here might be necessary. Titles of webpages should generally reflect their immediate contents. Consider something to the tune of "Post Tags" or "Blog Tags" for the title. For the paragraph, maybe something like:

> "Looking for a specific topic in our blog? Then you've come to the right place! ..."

may benefit the page.